### DISCUSSION TITLE
Clean install of Laravel 9.16.0 on PHP 8.1.6 -> Unexpected token in class loader

### DISCUSSION THREAD

**Discussion by sblindt** [laravel/framework#42697] [Q&A] [answered]
Hey everyone, 

Please see the stack trace below. I am easily able to reproduce on a fresh, clean install using Laravel 9 on PHP 8.1.  To date, I have not been able to use Laravel 9 on PHP 8.1.

[2022-06-07 11:34:36] laravel.EMERGENCY: Unable to create configured logger. Using emergency logger. {"exception":"[object] (InvalidArgumentException(code: 0): Log [] is not defined. at /vendor/laravel/framework/src/Illuminate/Log/LogManager.php:207)
[stacktrace]
#0 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Log/LogManager.php(132): Illuminate\\Log\\LogManager->resolve()
#1 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Log/LogManager.php(119): Illuminate\\Log\\LogManager->get()
#2 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Log/LogManager.php(702): Illuminate\\Log\\LogManager->driver()
#3 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(274): Illuminate\\Log\\LogManager->log()
#4 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(415): Illuminate\\Foundation\\Exceptions\\Handler->report()
#5 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(113): Illuminate\\Foundation\\Http\\Kernel->reportException()
#6 /home/vagrant/code/repo/public/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle()
#7 {main}
"} 

[2022-06-07 11:34:36] laravel.ERROR: syntax error, unexpected token ")" {"exception":"[object] (ParseError(code: 0): syntax error, unexpected token \")\" at /home/vagrant/code/repo/vendor/symfony/finder/Finder.php:588)
[stacktrace]
#0 /home/vagrant/code/repo/vendor/composer/ClassLoader.php(428): Composer\\Autoload\\includeFile()
#1 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php(88): Composer\\Autoload\\ClassLoader->loadClass()
#2 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php(65): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->getConfigurationFiles()
#3 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php(39): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->loadConfigurationFiles()
#4 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(239): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->bootstrap()
#5 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(153): Illuminate\\Foundation\\Application->bootstrapWith()
#6 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(137): Illuminate\\Foundation\\Http\\Kernel->bootstrap()
#7 /home/vagrant/code/repo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#8 /home/vagrant/code/repo/public/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle()
#9 {main}
"}

**Comment by sblindt** [comment, marked as answer]
FYI, this was resolved by updating from Homestead 11 to Homestead 12.

**Comment by dubel5** [comment]
PHP 8.1.0
9.6.0
syntax error, unexpected token ")", expecting identifier or variable or "{" or "$"
please cant somebody help me ?
